themes/legacy/assets/js/config/search.js

```diff
@@ -79,7 +79,9 @@ MEDUSA.config.search = function() {
 
             if (selectedProvider.toLowerCase() === 'utorrent') {
                 client = 'uTorrent';
-                $('#torrent_path_option').hide();
+                $('#label_warning_utorrent').show();
```

Comment:
We shouldn't be updating legacy at all.

Reply:
I wasn't aware, no problem to remove that change, let me know.